### PR TITLE
Added schemaCreation to posgres User func

### PR DESCRIPTION
**Pull request by YuriiBudnyi**

### CODE REVIEW

File: postgres/postgres.libsonnet [hunk 1]
```diff
@@ -39,7 +39,7 @@ local database(databaseName, hostName, dropOnDeletion=true, name='', extensions=
   },
 };
 
-local user(username, databaseName, priv, secretName='', name='') = {
+local user(username, databaseName, priv, secretName='', name='', schemaCreation='') = {
```
Comment (PatTheSilent) [inline postgres/postgres.libsonnet:42]:
```suggestion
local user(username, databaseName, priv, secretName='', name='', schemaCreation=false) = {
```

File: postgres/postgres.libsonnet [hunk 2]
```diff
@@ -63,6 +63,7 @@ local user(username, databaseName, priv, secretName='', name='') = {
       databaseRef: {
         name: databaseName
       },
+      [if schemaCreation != '' then 'schemaCreation']: schemaCreation,
```
Comment (PatTheSilent) [inline postgres/postgres.libsonnet:66]:
```suggestion
      schemaCreation: schemaCreation,
```

**Review by PatTheSilent — CHANGES_REQUESTED** [review]

File: postgres/postgres.libsonnet (hunk 1 above)
Comment (YuriiBudnyi) [reply postgres/postgres.libsonnet]: Done

File: postgres/postgres.libsonnet (hunk 2 above)
Comment (YuriiBudnyi) [reply postgres/postgres.libsonnet]: Done

**Review by PatTheSilent — APPROVED** [review]

**Pull request merged**